README.md

```diff
@@ -6,125 +6,160 @@
 
 ```js
-function sayHello(person) {
-  console.log('Hello, ' + person.name + '!');
+function Chair(style, color) {
+  console.log(this);
+  this.style = style;
+  this.color = color;
 }
+sofa = new Chair("sofa", "green");
 ```
 
-Let's also define a function for creating a person:
+Here, because we're calling `Chair()` with the `new` keyword, `this`
+refers to the new `Chair` object. So when you see `this.style = style;`
+you can think of it as saying "set the style property of THIS current
+instance of a Chair."
 
-```js
-function personMaker(name) {
-  this.name = name;
-
-  return this;
-}
-```
+In your console, `this` will be an empty Chair
+object because we haven't assigned the property values yet when we
+output `this`, but try moving that `console.log(this)` to the end of the
+function and see what you get!
 
-Now if we call `sayHello(personMaker('Alf'))`, we should see `Hello, Alf!`. Try it!
+### Object Function Call
 
-Let's take another example to highlight some of jQuery's magic. This function handle the `load` event.
+Objects can have functions as well as properties. When a function is an attribute of an object, `this` is set to the object instance that contains the function, similar to how it works with a constructor function.
 
-```html
-<img class="pix" src="dog.jpg">
-<img class="pix" src="cat.jpg">
-<img class="pix" src="pig.jpg">
-```
 
 ```js
-function frameIt(){
-  $('.pix').on('load', function() {
-    $('.pix').addClass("tasty");
-  });
-}
+var couch = {
+  color: 'green',
+  f: function() {
+    return this;
+  }
+};
+
+console.log(couch.f());
+// outputs object
 ```
 
-There is actually a really great way to refactor the above example using `this`. We're calling the `on` function on the class `pix`. Because of that fact, we have access to `this` inside of the function, and can replace part of`$('.pix').addClass("tasty");` with `this`:
+**Top-tip:** Notice that this only outputs an `Object` and not something called
+`couch`? That's because `this` is just an instance of the base `Object`
+type here, and `couch` is just the name of the variable holding that
+object instance. The object itself is the owner of the function, and has
+no idea what you named it.
 
-```js
-function frameIt(){
-  $('.pix').on('load', function() {
-    $(this).addClass("tasty");
-    alert("added class!");
-  });
-}
+### DOM Events
+
+We can also make use of `this` when handling events on the DOM. If our
+`index.html` has three image elements with the same css class, we might
+need to know which specific `img` was clicked.
+
+```html
+<img class="pix" src="http://i.giphy.com/S1phUc5mmaZqM.gif">
+<img class="pix" src="http://i.giphy.com/eGe59ekUJEll6.gif">
+<img class="pix" src="http://i.giphy.com/l41lNT5u8hCI92nQc.gif">
+<script type="text/javascript" charset="utf-8">
+    var els = document.getElementsByClassName("pix");
+    function handleClick(e) {
+      console.log(this);
+    }
+    for(i=0 ; i < els.length ; i++){
```

Comment:
thanks! not nearly as up on my js idioms as I am some other langs so please keep pointing out these nits!